src/commands/txCommands/pushTxMaster.js

```diff
@@ -0,0 +1,33 @@
+const chalk = require('chalk');
```

Comment:
nitpick: could this script be run as an option on `tx push`, e.g. `mope tx push --master`? From a conceptual point of view, I feel like `push` should be the umbrella command for all tx pushes, but you can opt-in to pushing in a special 'master' mode by supplying a `--master` argument.

Implementation-wise, you could just update the `handler` for `push` to check for `argv.master` and run this script instead of `pushContent$.subscribe(..)`. I think this script would then move into `txCommands/util` or `txCommands/scripts` or something similar